src/navigation/Nav.jsx

```diff
@@ -217,7 +217,9 @@ export class Nav extends React.Component {
 			self.is_pro_admin && {
 				shrink: true,
 				linkTo: media.isAtMediumUp ? proDashboard.link : '',
-				onAction: !media.isAtMediumUp && this.onClickMobileDropdownAction,
+				onAction:
```

Comment:
Would it make more sense to not include `onAction` here when `!media.isAtMediumUp` is false?